### PR TITLE
[Fix](Hive-Metastore)  fix that if JDBC reads the NULL value, it will cause NPE

### PR DESCRIPTION
**Pull request by BePPPower**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: fe/fe-core/src/main/java/org/apache/doris/datasource/hive/PostgreSQLJdbcHMSCachedClient.java [hunk 1]
```diff
@@ -470,6 +470,15 @@ private List<FieldSchema> getSchemaExcludePartitionKeys(int sdId) {
         return colsExcludePartitionKeys.build();
     }
 
+    private String getStringResult(ResultSet rs, String columnLabel) throws Exception {
+        String s = rs.getString(columnLabel);
+        if (rs.wasNull()) {
+            LOG.warn("get `NULL` value of field `" + columnLabel + "`.");
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/datasource/hive/PostgreSQLJdbcHMSCachedClient.java:476]: Change to debug

**Comment by BePPPower** [comment]
run buildall

**Comment by BePPPower** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

**Review by zy-kkk — APPROVED** [review]

**Pull request merged**